tests/PhenX.EntityFrameworkCore.BulkInsert.Tests/PhenX.EntityFrameworkCore.BulkInsert.Tests.csproj

```diff
@@ -13,8 +13,10 @@
           <PrivateAssets>all</PrivateAssets>
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
+        <PackageReference Include="FluentAssertions" Version="7.2.0" />
```

Comment:
I'm not very used to fluent assertions nor like the fluent syntax for this usage, was it necessary ?

Reply:
it is easier not to miss something. If you just test one property you can easily miss edge cases where your assertions are really bad.

Reply:
This was more a problem from the quality of my assertions, not the lack of fluent assertions.
That and the recent licensing issues with FluentAssertions, I was not ready for this.
Next time you want to integrate a new dependency I'd like to discuss about it.

Reply:
No problem, by the way I love Verify!